lizmap/modules/lizmap/controllers/service.classic.php

```diff
@@ -807,6 +858,7 @@ protected function GetFeature($wfsRequest)
         if (substr($result->data, 0, 7) == 'file://' && is_file(substr($result->data, 7))) {
             $rep->fileName = substr($result->data, 7);
             $rep->deleteFileAfterSending = true;
+            $rep->content = null;
```

Comment:
PHPStan:
```
Property jResponseBinary::$content (string) does not accept null. 
```

Reply:
the code is good. This is the comment of the `content` property which is not complete. I fixed it.